R/ci.R

```diff
@@ -2,7 +2,7 @@
 #'
 #' @description
 #'
-#' \lifecycle{soft-deprecated}
+#' `r lifecycle::badge("deprecated")`
 #'
 #' Some of these functions are now soft-deprecated since the tidyverse team has
```

Comment:
```suggestion
#' Some of these functions are now deprecated since the tidyverse team has
```

Reply:
Should we actually deprecate these, i.e. in the code? Right now the deprecation is just a state of mind.

Reply:
Soft deprecation is still a thing; it just doesn't get it's own badge; and it looks like `use_travis()` is already calling `deprecate_soft()`?